From: vishalk17 <vishalkapadi95@gmail.com>
Date: Fri, 17 Nov 2017 16:36:20 +0000
Subject: [PATCH] This will allow more than one directories of BOARD_PLAT_PUBLIC_SEPOLICY & BOARD_PLAT_PRIVATE_SEPOLICY

diff --git a/Android.mk b/Android.mk
index f4c5ab1d..0c28874c 100644
--- a/Android.mk
+++ b/Android.mk
@@ -81,19 +81,19 @@ endif
 
 PLAT_PUBLIC_POLICY := $(LOCAL_PATH)/public
 ifneq ( ,$(BOARD_PLAT_PUBLIC_SEPOLICY_DIR))
-ifneq (1, $(words $(BOARD_PLAT_PUBLIC_SEPOLICY_DIR)))
-$(error BOARD_PLAT_PUBLIC_SEPOLICY_DIR must only contain one directory)
-else
+#ifneq (1, $(words $(BOARD_PLAT_PUBLIC_SEPOLICY_DIR)))
+#$(error BOARD_PLAT_PUBLIC_SEPOLICY_DIR must only contain one directory)
+#else
 PLAT_PUBLIC_POLICY += $(BOARD_PLAT_PUBLIC_SEPOLICY_DIR)
-endif
+#endif
 endif
 PLAT_PRIVATE_POLICY := $(LOCAL_PATH)/private
 ifneq ( ,$(BOARD_PLAT_PRIVATE_SEPOLICY_DIR))
-ifneq (1, $(words $(BOARD_PLAT_PRIVATE_SEPOLICY_DIR)))
-$(error BOARD_PLAT_PRIVATE_SEPOLICY_DIR must only contain one directory)
-else
+#ifneq (1, $(words $(BOARD_PLAT_PRIVATE_SEPOLICY_DIR)))
+#$(error BOARD_PLAT_PRIVATE_SEPOLICY_DIR must only contain one directory)
+#else
 PLAT_PRIVATE_POLICY += $(BOARD_PLAT_PRIVATE_SEPOLICY_DIR)
-endif
+#endif
 endif
 PLAT_VENDOR_POLICY := $(LOCAL_PATH)/vendor
 REQD_MASK_POLICY := $(LOCAL_PATH)/reqd_mask
